### ISSUE TITLE
ijai.vacuum.v18, 2023.6.1, scripts doesn't work

### ISSUE THREAD

**Issue by lirtm** [al-one/hass-xiaomi-miot#1154]
Actually, subject.

I have script:
```
cleanhallroom:
  alias: Уборка коридора
  sequence:
  - service: xiaomi_miot.call_action
    data:
      entity_id: vacuum.ijai_v18_e134_robot_cleaner
      siid: 7
      aiid: 4
      params:
      - piid: 34
        value: '[ "1_12_0_2_0_0_1_1"]'
      - piid: 33
        value: 1682530583
  - delay:
      hours: 0
      minutes: 0
      seconds: 1
      milliseconds: 0
  - service: xiaomi_miot.call_action
    data:
      entity_id: vacuum.ijai_v18_e134_robot_cleaner
      siid: 7
      aiid: 3
      params:
      - '12'
      - 0
      - 1
  mode: single
  icon: mdi:sofa-single
```

Now, if I start script. I see this in logs:

```
Logger: homeassistant.components.script.cleanhallroom
Source: custom_components/xiaomi_miot/__init__.py:422
Integration: Скрипт (documentation, issues)
First occurred: 18:39:04 (1 occurrences)
Last logged: 18:39:04

Уборка коридора: Error executing script. Unexpected error for call_service at pos 1: Passing coroutines is forbidden, use tasks explicitly.
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 452, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 685, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1910, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1950, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/config/custom_components/xiaomi_miot/__init__.py", line 422, in async_service_handler
    await asyncio.wait(update_tasks)
  File "/usr/local/lib/python3.11/asyncio/tasks.py", line 415, in wait
    raise TypeError("Passing coroutines is forbidden, use tasks explicitly.")
TypeError: Passing coroutines is forbidden, use tasks explicitly.

```

```Эта ошибка возникла в пользовательской интеграции.

Logger: homeassistant
Source: custom_components/xiaomi_miot/__init__.py:422
Integration: Xiaomi Miot Auto (documentation, issues)
First occurred: 18:39:04 (1 occurrences)
Last logged: 18:39:04

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/script/__init__.py", line 477, in _async_run
    return await self.script.async_run(script_vars, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 1533, in async_run
    await asyncio.shield(run.async_run())
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 410, in async_run
    await self._async_step(log_exceptions=False)
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 454, in _async_step
    self._handle_exception(
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 477, in _handle_exception
    raise exception
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 452, in _async_step
    await getattr(self, handler)()
  File "/usr/src/homeassistant/homeassistant/helpers/script.py", line 685, in _async_call_service_step
    await service_task
  File "/usr/src/homeassistant/homeassistant/core.py", line 1910, in async_call
    task.result()
  File "/usr/src/homeassistant/homeassistant/core.py", line 1950, in _execute_service
    await cast(Callable[[ServiceCall], Awaitable[None]], handler.job.target)(
  File "/config/custom_components/xiaomi_miot/__init__.py", line 422, in async_service_handler
    await asyncio.wait(update_tasks)
  File "/usr/local/lib/python3.11/asyncio/tasks.py", line 415, in wait
    raise TypeError("Passing coroutines is forbidden, use tasks explicitly.")
TypeError: Passing coroutines is forbidden, use tasks explicitly.
```

```Эта ошибка возникла в пользовательской интеграции.

Logger: custom_components.xiaomi_miot.sensor
Source: custom_components/xiaomi_miot/sensor.py:702
Integration: Xiaomi Miot Auto (documentation, issues)
First occurred: 18:40:16 (1 occurrences)
Last logged: 18:40:16

Get xiaomi message for 1547084676 failed: {}```

```Эта ошибка возникла в пользовательской интеграции.

Logger: custom_components.xiaomi_miot.core.xiaomi_cloud
Source: custom_components/xiaomi_miot/core/xiaomi_cloud.py:209
Integration: Xiaomi Miot Auto (documentation, issues)
First occurred: 18:36:39 (10 occurrences)
Last logged: 18:43:03

Request xiaomi api: v2/message/v2/check_new_msg {'begin_at': 1686324926} failed, response: None
Request xiaomi api: v2/message/v2/typelist {} failed, response: None```

**Comment by Drealine** [comment]
Same problem here since update to HA 2023.6

**Comment by al-one** [comment]
Fixed in `v0.7.10`, please upgrade.
